src/patches/banks/upi-enabled-banks.yml

```diff
@@ -161,3 +161,16 @@ banks:
   - VVSB
   - XJKG
   - YESB
+  - VCCX
```

Comment:
validated these on the website? also order them in alphabetical;

Reply:
yes. @captn3m0 mentioned above that we need to keep all the names in case of a conflict like above. But if we do so, won't it increase the number of banks which have upi-enabled and fail the upi check `parse_upi` in `scraper/scripts/methods_nach.rb`.

Reply:
@patlola / @captn3m0 My doubt is, what bank code should we fill in `src/patches/banks/upi-enabled-banks.yml` in case of a conflict as mentioned here https://github.com/razorpay/ifsc/pull/188#issuecomment-689961991 . We can't keep both since the parse_upi function check in `scraper/scripts/methods_nach.rb` will fail.

Reply:
- `SDSX` vs `SDCE` -> We pick `SDCE`, since the one without the X is the more important one
- `CZCX` vs `CCBX` -> Have to check lineage of each. There might have been a merger involved.

Reply:
We have the following [non-scheduled co-operative banks called Citizen](https://rbidocs.rbi.org.in/rdocs/Content/pdfs/nonschedulecoop.pdf) Bank:

1. The Citizens Urban Co op Bank Limited Jallandhar
2. The Citizen Co operative Bank Limited (Delhi)
3.  Citizens' Urban Co-operative Bank Ltd (Gangtok)
4. The Citizens' Co-operative Bank Ltd., Jammu
5. Citizen Co-operative Bank Ltd. (Lucknow)
6. Citizen Co-op. Bank Ltd. Vasco-De-Gama.
7. Citizen Coop Bank Limited Rajkot.
8. The Citizen Co-operative Bank Ltd., Bangalore.

And another bank that had [its license revoked](https://www.rbi.org.in/scripts/BS_PressReleaseDisplay.aspx?prid=20770#):

- Citizen Co-operative Bank Ltd., Burhanpur

Reply:
And the following codes:

Code|Bank
-- | -- 
CCBX | THE CITIZEN COOPERATIVE BANK LIMITED |  
CJAX | THE CITIZENS' CO-OPERATIVE BANK LTD. JAMMU
CTBX | CITIZENS CO-OPERATIVE BANK LTD. |   |  
CURX | THE CITIZENS URBAN COOPERATIVE BANK LTD. |  
CZCX | CITIZEN CO OPERATIVE BANK LTD. |   |  

Reply:
Reached out to the bank for clarification :fingers_crossed:

Reply:
CCBX was picked.